apps/opik-frontend/.env.comet

```diff
@@ -1,4 +1,5 @@
 VITE_BASE_URL=/opik
 VITE_BASE_API_URL=/opik/api
+VITE_GET_STARTED_API_URL=/9b7f0af9b6f3aa8361bc52f44b9e7099
```

Comment:
don't you need this variable to any other `env` files? (not sure, just making sure)